### ISSUE TITLE
Issue saving offices on user profile

### ISSUE THREAD

**Issue by acory** [unicef/etools-issues#939]
After country change, if the Office on the user's profile is not on the current country there is a backend error on save user profile (Offices: pk '..' does not exist.)
Solution : On save , if office was not changed, do not send it to backend.

**Comment by robertavram** [comment]
good catch... @domdinicola .. if there is any place in the system that the user's office is being used, it is in Travel... if it's not used anywhere for any filters concerning users.. my proposal would be to remove the model and add it when we get a use case.. currently the model that is being used for Office was build a long time ago and needs to be deprecated or replaced with a better solution.

@domdinicola can you confirm where / if the office model is being used anywhere...

**Comment by domdinicola** [comment]
It's used... there are filters and is displayed as well in the list view.
Looking forward t2f decommission, we can think about removing it if the business agrees.

**Comment by robertavram** [comment]
ok.. @acory your solution can work for now... users don't change countries... when they do they should theoretically be mapped to the proper office.

**Comment by roxanavlad** [comment]
test result: pass
tested om demo env on Chrome and IE 11
